pkg/monitor/datapath_trace.go

```diff
@@ -57,8 +56,31 @@ type TraceNotify struct {
 	// data
 }
 
-// decodeTraceNotify decodes the trace notify message in 'data' into the struct.
-func (tn *TraceNotify) decodeTraceNotify(data []byte) error {
+// Dump prints the message according to the verbosity level specified
+func (tn *TraceNotify) Dump(args *api.DumpArgs) {
+	switch args.Verbosity {
+	case api.INFO, api.DEBUG:
+		tn.DumpInfo(args.Buf, args.Data, args.Format, args.LinkMonitor)
+	case api.JSON:
+		tn.DumpJSON(args.Buf, args.Data, args.CpuPrefix, args.LinkMonitor)
+	default:
+		fmt.Fprintln(args.Buf, msgSeparator)
+		tn.DumpVerbose(args.Buf, args.Dissect, args.Data, args.CpuPrefix, args.Format, args.LinkMonitor)
+	}
+}
+
+// GetSrc retrieves the sorce endpoint for the message.
+func (tn *TraceNotify) GetSrc() uint16 {
+	return tn.Source
+}
+
+// GetDst retrieves the destination endpoint for the message.
+func (tn *TraceNotify) GetDst() uint16 {
+	return tn.DstID
+}
```

Comment:
AFAIK for `TRACE_TO_PROXY` traces `DstID` is the proxy port, see [here](https://github.com/cilium/cilium/blob/1a6fbe49a84450cf528247d212796d335b8e2be8/bpf/lib/trace.h#L14) and [here](https://github.com/cilium/cilium/blob/1a6fbe49a84450cf528247d212796d335b8e2be8/bpf/bpf_lxc.c#L671-L673).

Reply:
Thank you! I left a small comment on the method